packages/desktop-client/src/components/schedules/ScheduleDetails.jsx

```diff
@@ -225,7 +225,9 @@ export function ScheduleDetails({ modalProps, actions, id, transaction }) {
               _account: transaction.account,
               _amount: transaction.amount,
               _amountOp: 'is',
-              name: transaction.payee ? payees[transaction.payee].name : '',
+              name: transaction.payee
+                ? payees[payees.findIndex(e => e.id === transaction.payee)].name
```

Comment:
`payees` is an array of objects so searching for the index then using that index to get the name fixed the bug I was seeing.  I don't understand JS enough to have an idea why it worked before.